code/__HELPERS/roundend.dm

```diff
@@ -285,6 +285,7 @@
 			var/list/ded = SSblackbox.first_death
 			if(ded.len)
 				parts += "[GLOB.TAB]First Death: <b>[ded["name"]], [ded["role"]], at [ded["area"]]. Damage taken: [ded["damage"]].[ded["last_words"] ? " Their last words were: \"[ded["last_words"]]\"" : ""]</b>"
+			//fukken syntax parsing memes"
```

Comment:
please don't leave comments that someone else will have to remove later

Reply:
It's not useless. The \" in the line directly above it isn't being parsed correctly, resulting in the rest of the file being fubar'd on visual code.
![image](https://user-images.githubusercontent.com/6209658/38639888-95f9f328-3da0-11e8-8f90-913a7ce0efff.png)

![image](https://user-images.githubusercontent.com/6209658/38639905-a5765f3a-3da0-11e8-9c79-176d652e66d2.png)

Reply:
welp